### ISSUE TITLE
Improve score of JSR symbol

### ISSUE THREAD

**Issue by fzn0x** [honojs/hono#2861]
### What is the feature you are proposing?

![Screenshot 2024-05-30 064618](https://github.com/honojs/hono/assets/50759463/a9887bf4-48d0-424b-a9ce-c133e7389d79)

Reference: https://jsr.io/docs/writing-docs#symbol-documentation

Now (when this issue was created) we only have 19% of the code documented, we can start with the public methods first before focusing on the private ones. In future, after we done with the JSR symbol, we can continously improve the code documentation.

**Comment by EdamAme-x** [comment]
Progress
https://github.com/honojs/hono/pull/2836

**Comment by fzn0x** [comment]
Lets get 80% ! 👍

**Comment by yusukebe** [comment]
Regarding the JSR score, I think it's okay to cover only public APIs. "comment" is not always good for private methods or properties. Perhaps you may find the spec by looking at the code or test, not the comment. Honestly, I don't like over-writing the comment. So, I think it's enough to write docs for the API marked as `export`, and we shouldn't be eager to get 100 points.

**Comment by EdamAme-x** [comment]
> Regarding the JSR score, I think it's okay to cover only public APIs. "comment" is not always good for private methods or properties. Perhaps you may find the spec by looking at the code or test, not the comment. Honestly, I don't like over-writing the comment. So, I think it's enough to write docs for the API marked as `export`, and we shouldn't be eager to get 100 points.

I agree. If we should be eager to get 100 points, it is too difficult to maintenance code.

**Comment by fzn0x** [comment]
> I think it's enough to write docs for the API marked as `export`, and we shouldn't be eager to get 100 points.

Yes, I think public APIs alone is enough to get 80 points.

**Comment by EdamAme-x** [comment]
Went up 3% by https://github.com/honojs/hono/pull/2836
The main API is now mostly jsdoc added.

![image](https://github.com/honojs/hono/assets/121654029/4ab6d8be-13fc-43dd-967d-0d12645dbda8)

**Comment by yusukebe** [comment]
@HeyITGuyFixIt 

Thanks for the information! Currently, the score of `@hono/hono` is `76%`. It's good to be higher but I think we don't have to be eager for it. The user will use Hono if it's not a bad score.

![CleanShot 2024-10-31 at 11 04 34@2x](https://github.com/user-attachments/assets/7496e048-5a50-4504-b9f2-957fce6b8499)
